package/MDAnalysis/core/universe.py

```diff
@@ -238,6 +238,18 @@ class Universe(object):
     vdwradii: dict, ``None``, default ``None``
         For use with *guess_bonds*. Supply a dict giving a vdwradii for each
         atom type which are used in guessing bonds.
+    context: string or Guesser, default ``default``
+        Type of the Guesser to be used in attributes guessing
```

Comment:
```suggestion
        Type of the Guesser to be used in guessing TopologyAttrs
```

Just to be a bit more precise